delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/routes/ElasticSearchViewsDirectives.scala

```diff
@@ -102,6 +105,7 @@ trait ElasticSearchViewsDirectives extends UriDirectives {
           updatedBy,
           updatedAt,
           types,
+          typeOperator.getOrElse(AND),
```

Comment:
It's a little unfortunate this duplication (both in this file and the default argument), but I wasn't sure of a better way